silverstripe/cms/ss-2017-004.yaml

```diff
@@ -0,0 +1,14 @@
+title: 'SS-2017-004: XSS in page history comparison'
+link: 'https://www.silverstripe.org/download/security-releases/ss-2017-004'
+cve: '~'
+branches:
+    3.4.x:
+        time: '2017-06-02 01:55:25'
+        versions: ['>=3.4.0', '<3.4.6']
+    3.5.x:
+        time: '2017-06-02 01:55:25'
+        versions: ['>=3.5.0', '<3.5.4']
+    3.x:
+        time: '2017-06-02 01:55:25'
+        versions: ['>=3.4.0', '<3.6.0']
```

Comment:
This should just read `3.6.0` for this last one - and probably indicate `3.6` as the branch too.